test/integration_test.js

```diff
@@ -443,6 +443,10 @@ describe("Integration tests", function() {
             expect(syncResult.lastModified).to.be.a("number");
           });
 
+          it("should have updated lastModified", () => {
+            expect(syncResult.lastModified).to.equal(tasks.lastModified);
```

Comment:
We should also check for the collection's `lastModified` value stored in IndexedDB (it's the actual one we use to fetch the next chunk of changes, if any).